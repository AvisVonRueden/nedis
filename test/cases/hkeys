*6
$5
hmset
$4
user
$4
name
$2
tj
$5
email
$4
tobi
--------
*2
$5
hkeys
$4
user
