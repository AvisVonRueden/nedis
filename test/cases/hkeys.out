+OK
*2
$4
name
$5
email
